php/class-plugin.php

```diff
@@ -163,6 +169,9 @@ function init() {
 		if ( $this->is_module_active( 'deferred_customize_widgets' ) ) {
 			$this->deferred_customize_widgets = new Deferred_Customize_Widgets( $this );
 		}
+		if ( $this->is_module_active( 'customize_settings_snapshot' ) ) {
+			$this->customize_settings_snapshot = new Customize_Settings_Snapshot( $this );
```

Comment:
`Customize_Snapshot_Manager` would be doing the same things for `Customize_Settings_Snapshot` that `WP_Customize_Manager` does for `WP_Customize_Transaction`. In other words, the manager class would instantiate the `Customize_Settings_Snapshot` class, it would pass values into it via Ajax and get values out of it. The `Customize_Settings_Snapshot` would be standalone and more agnostic to its environment, like `WP_Customize_Setting` is.